### PR TITLE
Add processes count to /stats

### PR DESCRIPTION
**Pull request by ismaelga**
Also removed unused `each_with_index`. Tell me if there is any reason to keep them or if I should remove it from the PR.

:yellow_heart:

### CODE REVIEW

**Comment by seuros** [comment]
Please add an entry to the Changelog

File: lib/sidekiq/web_helpers.rb [hunk 1]
```diff
@@ -46,6 +46,14 @@ def workers
       @workers ||= Sidekiq::Workers.new
     end
 
+    def processes_size
```
Comment (mperham) [inline lib/sidekiq/web_helpers.rb]: I don't think this method is necessary.

**Comment by mperham** [comment]
Thanks for the cleanup, this is nice.  I should probably go through the Stats call and optimize the Redis usage.  We're probably making too many roundtrips to gather all the different stats.

File: lib/sidekiq/web_helpers.rb (hunk 1 above)
Comment (ismaelga) [inline lib/sidekiq/web_helpers.rb]: I agree.

**Comment by ismaelga** [comment]
Are you thinking on using pipelined to reduce the roundtrips?

**Comment by mperham** [comment]
Exactly.  Merge all the operations to gather the stats into a single pipelined block.

**Comment by mperham** [comment]
If you want to do that, feel free.  It'd be a welcome optimization.

**Comment by ismaelga** [comment]
Sure. I'll have a look and send another PR.

**Comment by mperham** [comment]
I'll take this as is and polish it up a bit.

**Comment by ismaelga** [comment]
Cool. I'm close to send the other PR

**Pull request merged**